client/my-sites/posts/post-controls.jsx

```diff
@@ -64,7 +64,7 @@ const getAvailableControls = props => {
 
 		controls.main.push( {
 			className: 'stats',
-			href: `/stats/post/${ post.ID }/${ site.slug }`,
+			href: `/stats/post/${ post.ID }/${ siteSlug }`,
```

Comment:
While testing I'm going over `http://calypso.localhost:3000/posts/my` and the Stats link leads to `http://calypso.localhost:3000/stats/post/72/null` for some of them. It happens for some Jetpack sites and some WordPress.com sites too. Confirmed: this does not happen in production. Site slug is `null` for some reason, which does not make much sense. After further checking, the site with that `siteId` seems to not exist in state at all for some reason. Any ideas?